docs/usage/shoot-operations/shoot_updates.md

```diff
@@ -109,6 +133,113 @@ Changes to `kubeReserved` or `systemReserved` do not trigger a node roll if thei
 Generally, the provider extension controllers might have additional constraints for changes leading to rolling updates, so please consult the respective documentation as well.
 In particular, if the feature gate `NewWorkerPoolHash` is enabled and a worker pool uses the new hash, then the `providerConfig` as a whole is not included. Instead only fields selected by the provider extension are considered.
 
+### InPlace updates
+
+For the scenarios where user would like to keep the current node and avoid deletion of such nodes even after the updates, Gardener comes with the option of `in-place` update. Currently `in-place` update is kept behind `InPlaceNodeUpdates` feature gate.
+
+Machine image version in the cloudprofile has the `inPlaceUpdates` field which contains the configuration for in-place updates. For an OS to support in-place update `inPlaceUpdates.supported` field should be set to true. Moreoever `inPlaceUpdates.minVersionForUpdate` specifies the minimum supported version from which an in-place update to this machine image version can be performed.
+
+```yaml
+machineImages:
+- name: gardenlinux
+  updateStrategy: minor
+  versions:
+  - architectures:
+    - amd64
+    - arm64
+    classification: preview
+    cri:
+    - containerRuntimes:
+      - type: gvisor
+      name: containerd
+    version: 1632.0.0
+    inPlaceUpdates:
+      supported: true
+      minVersionForUpdate: 1630.0.0
+```
+
+#### InPlace Update Triggers
+
+An in-place update of the shoot worker nodes is triggered for rolling update triggers listed under [Rolling Update Triggers](#rolling-update-triggers) except the following:
+* `.spec.provider.workers[].machine.image.name`
+* `.spec.provider.workers[].machine.type`
+* `.spec.provider.workers[].volume.type`
+* `.spec.provider.workers[].volume.size`
+* `.spec.provider.workers[].cri.name`
+* `.spec.systemComponents.nodeLocalDNS.enabled`
+
+#### Validations for In-Place update
+
+Validations are in place to restrict any further updates when the current update is going on.
+For the cases where in-place update fails and the nodes are in problematic state, it requires user intervention to fix the node manually . After the fix user should be able to restart the update by the force update annotation `gardener.cloud/operation=force-in-place-update`. Without this annotation, any subsequent updates to the same worker pool are denied, refer [Force-update a worker pool with InPlace update strategy](shoot_operations.md#force-update-a-worker-pool-with-inplace-update-strategy) on how to force an update for worker pools.
```

Comment:
```suggestion
When a worker pool is undergoing an in-place update, applying subsequent updates to the same worker pool is restricted.
For the cases where in-place update fails and the nodes are in the failed state, it might require an operator intervention to fix the node. To update a failed worker pool to a new version, let's say, a new machine image patch version,  they can use the force in-place update annotation, refer [Force-update a worker pool with InPlace update strategy](shoot_operations.md#force-update-a-worker-pool-with-inplace-update-strategy) on how to force an update for worker pools.
```